src/WebJobs.Script/Metrics/HostMetrics.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Diagnostics.Metrics;
```

Comment:
On line 19 we set CloudPlatformName, should this be updated to use 

OpenTelemetryConstants.AzureCloudProviderValue

Reply:
I will change this in next PR.